src/modules/airspeed_selector/airspeed_selector_main.cpp

```diff
@@ -348,8 +345,11 @@ AirspeedModule::Run()
 				input_data.airspeed_timestamp = airspeed_raw.timestamp;
 				input_data.air_temperature_celsius = airspeed_raw.air_temperature_celsius;
 
+				float airspeed_stall = 7.0f;
+				param_get(param_find("FW_AIRSPD_STALL"), &airspeed_stall);
```

Comment:
@sfuhrer Can't we use ModuleParams here? Anyway, this should not be called every loop.

Reply:
64019907d170ca7e0c7cd8f8292c1941c978f8fc